# try... except block
try:
    my_num=input("Enter an integer : ")
    new_num=int(my_num)
    print(new_num)
except:
    print("Invalid task")